Published weekly changelog (#371)
CoboGlobal/product-manual: en/portal/changelog.mdx

description: "Stay updated with the latest documentation changes for Cobo Portal."
---

<Update label="2025/01/08 - 2025/01/14">
**Token Management**
- Revamped [Self-service token listing](/en/portal/self-service-token-listing) documentation and updated list of supported blockchains
- Added new [Supported tokens and chains](/en/portal/supported-tokens-and-chains) documentation

**Cobo Guard**
- Updated [Key features](/en/guard/key-features) with latest functionality
- Enhanced setup instructions in [Set up Cobo Guard](/en/guard/set-up)
- Updated key share backup process in [Generate and back up key shares](/en/guard/generate-and-back-up-key-shares)

> - Revised reset procedure in [Reset Cobo Guard](/en/guard/reset)

Reviewer: ```suggestion
- Revised the UI screenshots and relevant instructions in [Reset Cobo Guard](/en/guard/reset)
```

**Fee Station**

> - Enhanced [Fee Station overview](/en/portal/fee-station/overview) documentation

Author: ```suggestion
- Updated [Fee Station overview](/en/portal/fee-station/overview) documentation
```

- Updated [Set up Fee Station](/en/portal/fee-station/set-up) guide

**Staking**
- Updated ETH staking [Fee structure](/en/apps/staking/fee-structure) documentation

**Earn**

> - Added comprehensive documentation for the new Earn feature:

Reviewer 2: ```suggestion
- Added comprehensive documentation for the new Earn app:
```

  - [Introduction to Earn](/en/apps/earn/introduction)

>   - [Earn overview](/en/apps/earn/overview)

Reviewer 2: ```suggestion
  - [Overview](/en/apps/earn/overview)
```

  - [Set up Earn](/en/apps/earn/set-up)

>   - [Subscribe to Earn products](/en/apps/earn/subscribe)

Reviewer 2: ```suggestion
  - [Subscribe](/en/apps/earn/subscribe)
```

  - [View investment positions and activity](/en/apps/earn/view)

>   - [Redeem from Earn products](/en/apps/earn/redeem)

Reviewer 2: ```suggestion
  - [Redeem](/en/apps/earn/redeem)
```
